operator/cmd/mesh/install.go

```diff
@@ -271,3 +278,47 @@ func GetTagVersion(tagInfo string) (string, error) {
 	}
 	return tag.String(), nil
 }
+
+// GetProfileAndEnabledComponents get the profile and all the enabled components
+// from the given input files and --set flag overlays.
+func getProfileAndEnabledComponents(setOverlay []string, inFilenames []string, force bool,
+	kubeConfigPath string, context string, l clog.Logger) (string, []string, error) {
+	overlayYAML, profile, err := manifest.ReadYamlProfile(inFilenames, setOverlay, force, l)
+	if err != nil {
+		return "", nil, fmt.Errorf("failed to read profile: %v", err)
+	}
+	kubeConfig, _, err := InitK8SRestClient(kubeConfigPath, context)
```

Comment:
Is it possible to use `kubeClient` from `runApplyCmd` instead of creating new instance here?

Reply:
Yes, the `pre-check` is executed before the actual `InstallManifests`, we can init the `kubeClient` from `pre-check` and  reused it in `InstallManifests`.

Reply:
Done.

Reply:
I think you can use `kubeClient` from here?

https://github.com/istio/istio/blob/8473b663bc67280966bb40abbde7766abb1374eb/operator/cmd/mesh/install.go#L133-L136